docs/v0.7/en/reference/sql/range.md

```diff
@@ -60,74 +63,135 @@ ALIGN '5s' BY (host) FILL PREV;
 | `LINEAR` | Use [linear interpolation](https://en.wikipedia.org/wiki/Linear_interpolation) to fill the data. If an integer type is filled with `LINEAR`, the variable type of the column will be implicitly converted to a floating point type during calculation |
 |   `X`    |                                           Fill in a constant, the data type of the constant must be consistent with the variable type of the Range expression                                            |
 
+:::tip NOTE
+Note that if there are multiple Range expressions and the FILL method is specified for only one of them, in order to keep the number of SQL output rows uniform, the other Range expressions will use the FILL NULL method to fill in the missing time slots. So the following two SQL statements are equivalent in output:
+
+```sql
+SELECT 
+    ts, 
+    host, 
+    min(cpu) RANGE '10s',
+    max(cpu) RANGE '10s' FILL LINEAR 
+FROM host_cpu 
+ALIGN '5s';
+```
+
+equal to
+
+```sql
+SELECT 
+    ts, 
+    host, 
+    min(cpu) RANGE '10s' FILL NULL,
+    max(cpu) RANGE '10s' FILL LINEAR 
+FROM host_cpu 
+ALIGN '5s';
+```
+:::
 
 Take the following table as an example:
 
 ```sql
 +---------------------+-------+------+
-| ts                  | host  | cpu  |
+| ts                  | host  | val  |
```

Comment:
If you are using `val` as the column name, please rename all `cpu` column to `val` column in the section of the `FILL` option, including SQLs and results.